.github/workflows/verify-submodule-pull-request.yml

```diff
@@ -16,3 +16,7 @@ jobs:
       uses: Lombiq/GitHub-Actions/.github/actions/verify-submodule-pull-request@dev
       with:
         title: ${{ github.event.pull_request.title }}
+    - name: Check for Merge Conflict in PR
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Add%20a%20line%20break%20between%20two%20job%20steps